napari/layers/image/image.py

```diff
@@ -249,93 +253,96 @@ def __init__(
         # Trigger generation of view slice and thumbnail
         self._update_dims()
 
-    def _calc_data_range(self):
+    def _calc_data_range(self) -> Tuple[float, float]:
         if self.multiscale:
             input_data = self.data[-1]
         else:
             input_data = self.data
         return calc_data_range(input_data)
 
     @property
-    def dtype(self):
-        return self.data[0].dtype if self.multiscale else self.data.dtype
+    def dtype(self) -> np.dtype:
+        if self.multiscale:
+            return self.data[0].dtype
+        else:
+            return cast(ArrayLike, self.data).dtype
 
     @property
-    def data(self):
-        """array: Image data."""
+    def data(self) -> Union[ArrayLike, List[ArrayLike]]:
+        """Image data. Will be a list of arrays if self.multiscale"""
```

Comment:
```suggestion
        """Image data. Will be a list of arrays if self.multiscale is True."""
```

This maybe looks like the sentence has been accidentally truncated, this might help make it clear that's not the case.